### PR TITLE
add entra id doc

### PR DESCRIPTION
**Pull request by RazcoDev**

### CODE REVIEW

File: docs/integrations/SCIM/EntraID.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+### Connecting SCIM with Permit.io
+
+This guide will walk you through integrating SCIM with Permit.io using Entra ID (Formerly Azure Active Directory). SCIM (System for Cross-domain Identity Management) allows you to manage user identities seamlessly across systems.
+
+### Prerequisites
+
+Before you start, ensure you have the following:
+
+- Permit API Key
+- Permit's project ID and environment ID
```
Comment (obsd) [inline docs/integrations/SCIM/EntraID.mdx:16]: Add link to how to get it

File: docs/integrations/SCIM/EntraID.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+### Connecting SCIM with Permit.io
+
+This guide will walk you through integrating SCIM with Permit.io using Entra ID (Formerly Azure Active Directory). SCIM (System for Cross-domain Identity Management) allows you to manage user identities seamlessly across systems.
+
+### Prerequisites
+
+Before you start, ensure you have the following:
+
+- Permit API Key
```
Comment (obsd) [inline docs/integrations/SCIM/EntraID.mdx:15]: Libk

File: docs/integrations/SCIM/EntraID.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+8. Enter the following details:
+
+**Tenant URL:** `https://scim.permit.io/scim/v2/{permit_project_id}/{permit_env_id}`
+- **Optional**: base URL for **EU users** `https://scim.eu-central-1.permit.io/scim/v2/{permit_project_id}/{permit_env_id}`.
+
+:::info
+Replace `{permit_project_id}` and `{permit_env_id}` with your Permit project ID/Key and environment ID/Key respectively
+:::
+
+**Secret Token:** Your Permit API Key - Can be found in the Permit dashboard
```
Comment (obsd) [inline docs/integrations/SCIM/EntraID.mdx:55]: We have a doc page, link to there

**Review by obsd — APPROVED** [review]

File: docs/integrations/SCIM/EntraID.mdx (hunk 1 above)
Comment (RazcoDev) [reply docs/integrations/SCIM/EntraID.mdx]: addded

File: docs/integrations/SCIM/EntraID.mdx (hunk 2 above)
Comment (RazcoDev) [reply docs/integrations/SCIM/EntraID.mdx]: addded

File: docs/integrations/SCIM/EntraID.mdx (hunk 3 above)
Comment (RazcoDev) [reply docs/integrations/SCIM/EntraID.mdx]: added

**Pull request merged**